docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -100,7 +100,7 @@ function AppLayoutDocs(props) {
   }
 
   return (
-    <AppFrame className={'exclude-docsearch-indexing'}>
+    <AppFrame>
```

Comment:
I have searched in the Algolia docs, codebase, and Algolia crawler configuration, this class doesn't seem used, so removed it.